### PR TITLE
added beginner-friendly resources in the devops and python folder

### PR DESCRIPTION
**Pull request by Nkoli**

### CODE REVIEW

File: devops/README.md [hunk 1]
```diff
@@ -59,3 +59,11 @@ Your script will grab devopsshared_0*_file1.txt, file3.txt and file5.txt, then i
 
 # Resources
 TBD
```
Comment (kosyfrances) [inline devops/README.md]: @Nkoli You can remove line 61 since you have content added already.

File: python/README.md [hunk 2]
```diff
@@ -83,3 +83,7 @@ The program will first randomly generate a number unknown to the user. The user
 * [How to think like a Computer Scientist](http://interactivepython.org/runestone/static/thinkcspy/index.html)
 
 * [Real Python](https://realpython.com/)
+
+* [Programming With Mosh](https://www.youtube.com/watch?v=_uQrJ0TkZlc)
+
+* [ProgrammingKnowledge](https://www.youtube.com/watch?v=bZ6NL59FMoc)
```
Comment (kosyfrances) [inline python/README.md]: Are these python specific?

Comment (Nkoli) [reply python/README.md]: Yes they are.

File: devops/README.md (hunk 1 above)
Comment (Nkoli) [reply devops/README.md]: Done.

**Review by kosyfrances — APPROVED** [review]

**Pull request merged**